app/controllers/admin/events_controller.rb

```diff
@@ -23,11 +23,15 @@ def index
       @tracks_distribution_confirmed = @conference.tracks_distribution(:confirmed)
       @event_distribution = @conference.event_distribution
       @scheduled_event_distribution = @conference.scheduled_event_distribution
+      @pdf_filename = "#{@conference.title}.pdf"
```

Comment:
I would make the filename a bit more specific, like:
events_for_my_conference.pdf